FantasmoSDK/Classes/Network/FMApi.swift

```diff
@@ -170,17 +172,17 @@ class FMApi {
     /// - Parameters:
     ///   - frame: Frame to localize
     ///   - Returns: Formatted localization parameters
-    private func getParams(frame: ARFrame) -> [String : String] {
+    private func requestParams(for frame: ARFrame, with deviceOrientation: UIDeviceOrientation) -> [String : String] {
```

Comment:
If you don't like `getParams` maybe we should go with `params(for`. "Request" implies a remote async operation, in my mind anyway.

Reply:
Yes, "request" here is perceived as verb rather than as noun. I have changed it to "paramsOfLocalizeRequest".

"Localize" here is verb also, but we use it as common noun denoting including name of the request itself.

"get" prefix is used in Swift (and Ojbc as well) when one or more parameters are returned indirectly. For example:

```getParams(frame: ARFrame, inout params: [String : String])```